### PR TITLE
OKTA 794553 - Acrolinx edit for Inline hooks

### PR DESCRIPTION
**Pull request by thomascavanagh-okta**
## Description:
- **What's changed?** Improved the Acrolinx score for the [Inline hooks page](https://developer.okta.com/docs/concepts/inline-hooks/)
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-794553](https://oktainc.atlassian.net/browse/OKTA-794553)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md](https://github.com/okta/okta-developer-docs/blob/dd2e7e7f4e1b8775e736b4a677b12fa178640870/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Finline-hooks%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/10c144e0-25ff-40a1-a04b-3b6ad87befea) | 64  | 75  | 94  | 98  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.2596cc5b-6f6a-4a9f-8eb5-855f34c05d9b?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md [hunk 1]
```diff
@@ -130,17 +128,17 @@ Don't use the HTTP status code to return information to Okta regarding problems
 
 You can include any of the following types of objects in the JSON payload:
 
-#### commands
+#### Commands object
 
-Allows you to return commands to Okta to affect the process flow being executed and to modify values within Okta objects. The available commands differ by inline hook type and are defined in the specific documentation for each inline hook type.
+It allows you to return commands to Okta to affect the process flow being executed and to modify values within Okta objects. The available commands differ by inline hook type and are defined in the specific documentation for each inline hook type.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md:133]:
```suggestion
This allows you to return commands to Okta to affect the process flow being executed and to modify values within Okta objects. The available commands differ by inline hook type and are defined in the specific documentation for each inline hook type.
```

File: packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md [hunk 2]
```diff
@@ -130,17 +128,17 @@ Don't use the HTTP status code to return information to Okta regarding problems
...
 
 The `commands` object is an array, which allows you to return more than one command in your response. Each element within the array needs to consist of a pair of `type` and `value` elements. Each `type` element needs to be the name of a supported command you want to invoke. The corresponding `value` element is the operand that you want to specify for the command.
 
 The names of the commands follow Java-style reverse DNS name format, beginning with `com.okta`, followed by an Okta object that the command operates on, and then an action.
 
-#### error
+#### Error object
 
-Allows you to return error messages. How the error data is used varies by inline hook type.
+It allows you to return error messages. How the error data is used varies by inline hook type.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md:141]:
```suggestion
This allows you to return error messages. How the error data is used varies by inline hook type.
```

File: packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md [hunk 3]
```diff
@@ -163,17 +161,17 @@ An `ErrorCause` object must include the following fields:
 
 While there are no technical restrictions on the values for any of the fields in an `ErrorCause` object, using them as described in the previous table allows you to provide rich error information that can be useful in determining why an inline hook's processing failed.
 
-#### debugContext
+#### debugContext object
 
-Allows you to specify additional information to make available in the Okta System Log with the call to your hook. You can use this object as you want, sending any information that would be useful for debugging purposes. In the System Log, the content sent in this object is populated into the `inline_hook.response.processed` event.
+It allows you to specify additional information to make available in the Okta System Log with the call to your hook. You can use this object as you want, sending any information that would be useful for debugging purposes. In the System Log, the content sent in this object is populated into the `inline_hook.response.processed` event.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md:166]:
```suggestion
This allows you to specify additional information to make available in the Okta System Log with the call to your hook. You can use this object as you want, sending any information that would be useful for debugging purposes. In the System Log, the content sent in this object is populated into the `inline_hook.response.processed` event.
```

**Review by grahamsmith-okta — DISMISSED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md](https://github.com/okta/okta-developer-docs/blob/0d07634a3bc5533db9d652e3700ca2ce6bae4c55/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Finline-hooks%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5ece7428-f002-48cc-b53b-c06b204c1cf4) | 64  | 75  | 94  | 98  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.d1f0152a-23db-48b5-93e7-35ca2d2bcccb?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by grahamsmith-okta — APPROVED** [review]

**Pull request merged**